src/hevm/src/EVM/Expr.hs

```diff
@@ -53,21 +53,23 @@ sub = op2 Sub (-)
 mul :: Expr EWord -> Expr EWord -> Expr EWord
 mul = op2 Mul (*)
 
+-- TODO: should this return 0 on div by 0 or revert?
 div :: Expr EWord -> Expr EWord -> Expr EWord
-div = op2 Div Prelude.div
+div = op2 Div (\x y -> if y == 0 then 0 else Prelude.div x y)
 
+-- TODO: should this return 0 on div by 0 or revert?
```

Comment:
```suggestion

```

The behaviour with 0 divisor is also correct here now: https://www.evm.codes/#05?fork=arrowGlacier